src/MetaLoader.php

```diff
@@ -649,16 +649,42 @@ public function writeMetadataSerialize(string $outputDir): void
             foreach ($elements as $m) {
                 $entityId = $m['metadata']['entityid'];
 
-                Logger::debug(
-                    'metarefresh: Add metadata entry ' .
-                    var_export($entityId, true) . ' in set ' . var_export($set, true) . '.'
-                );
+                Logger::debug(sprintf(
+                    'metarefresh: Add metadata entry %s in set %s.',
+                    var_export($entityId, true),
+                    var_export($set, true),
+                ));
                 $metaHandler->saveMetadata($entityId, $set, $m['metadata']);
             }
         }
     }
 
 
+    /**
+     * This function uses the `PDO` metadata handler to upsert metadata in database.
+     * 
+     * @author - Tamilselvan Sekar, University of California, Davis. <tsekar@ucdavis.edu>
```

Comment:
Although your contribution is appreciated, please do not add @author tags. Someone may make a change to the method later etc. The git history will record who added which code when.